.gitlab-ci.yml

```diff
@@ -17,6 +17,9 @@ variables:
   CARGO_TARGET_DIR:                "/ci-cache/${CI_PROJECT_NAME}/targets/${CI_COMMIT_REF_NAME}/${CI_JOB_NAME}"
   RUST_LIB_BACKTRACE:              "0"
 
+  # Necessary for building binaryen-sys, which is part of the binaryen dependency
+  CXX:                             "/usr/bin/clang++-8"
```

Comment:
are you sure you want to put it on the top level? Means everything will be linked anew now. Similarly, we could put it in the CI image.

Reply:
I'm not sure about possible side-effects it would have in the CI image for other projects using the image.

Regarding top-level: It's needed for all the stages in the file (`test`, `build`, `fmt`), so yeah linking anew is needed for all of them.

Reply:
It's just ink that uses `ink-ci-linux` image currently. So yeah, if yoou need it everywhere, and if you'd use the CI image as a build local env (as it's possible with vscode), and maybe if someone compiles ink! externally, that'd make sense to introduce this variable to an image.

Reply:
Good point, I created https://github.com/paritytech/scripts/pull/235.